packages/appeals-service-api/src/routes/v2/appellant-submissions/_id/address/repo.js

```diff
@@ -24,26 +24,41 @@ class SubmissionAddressRepository {
 	/**
 	 * Create submission address for a given questionnaire
 	 *
-	 * @param {string} id
+	 * @param {string} appellantSubmissionId
 	 * @param {AddressData} addressData
 	 * @returns {Promise<AppellantSubmission>}
 	 */
-	async createAddress(id, addressData) {
+	async createAddress(appellantSubmissionId, addressData) {
```

Comment:
this won't work if there are any other address questions in an appellant journey or we add an add more address question at any point
I think you might need to potentially send the address id through in order to handle updates